### DISCUSSION TITLE
New Release

### DISCUSSION THREAD

**Discussion by ahankinson** [music-encoding/sibmei#203] [General]
Hi everyone!

If we can get #201 merged, then I think we should cut a new release of the plugin. Thoughts?

**Comment by rettinghaus** [comment]
About time after 18 months. 😉
This will be 4.1.0 then, correct?

**Comment by ahankinson** [comment]
Sure -- sounds good to me.

**Comment by th-we** [comment]
I was working on some [more substantial changes](https://github.com/music-encoding/sibmei/compare/aad512f595e8723530d1a78f24c53c26198aa45c...notengrafik:sibmei:wip/rewrite-layer-containers?expand=1), rewriting how layers, beams and tuplets are generated to make the state required for that less scattered and hard to work with.  That would fix some old bugs.  I hoped to get that working into the next release, but right now it does not seem like I have enough capacity to finish it soon.  So I guess, that will have to wait for the next release.

**Reply by ahankinson** [reply]
I think it's probably good to get it out now, while it's relatively "stable", and then your cleanup and refactor can happen as you have time for it.

**Comment by ahankinson** [comment]
I've just submitted PR #205 to fix #192, and merged an outstanding PR. Once that PR is approved and merged I think we can create the new release. Sound OK?
